src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -53,9 +55,18 @@ private static FormatConfig readConfig() {
         ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
 
         try {
-            return mapper.readValue(configFile, FormatConfig.class);
+            FormatConfig formatConfig = mapper.readValue(configFile, FormatConfig.class);
+            String repoFormat = formatConfig.getRepository();
+            if (repoFormat.matches("^[A-z,0-9].*/[A-z,0-9,-].*$") && repoFormat.split("/").length == 2) {
+                return formatConfig;
+            }
+            else
+                throw new BadRepositoryFormatException("Bad repository format in configuration file");
```

Comment:
do you think this check should be here? what if more checks will be added in the future. It would be useful to define some kind of external validation